### PR TITLE
preserve relation when a deleted model still has twins

### PR DESCRIPTION
**Pull request by vrcAlbert**

### CODE REVIEW

File: framework/classes/orm/twinnable/manymany.php [hunk 1]
```diff
@@ -219,4 +219,22 @@ public function _selectFallback($table, $table_fallback)
             'primary_key' => $primary_key,
         );
     }
+    
+    public function delete_related($model_from)
+    {
+        //search for twin models
+        $query = \DB::select()->from($this->table_through);
+        reset($this->key_from);
+        foreach ($this->key_through_from as $key)
```
Comment (felixgilles) [inline framework/classes/orm/twinnable/manymany.php]: Code styling issue : foreach bracket must be on the same line

**Comment by felixgilles** [comment]
A great PR ! Not trivial.

Fix the code styling issue and I merge it.

**Comment by vrcAlbert** [comment]
Done!
The wrong coding style came from parent method.
Another copy/paste issue :)

**Comment by felixgilles** [comment]
I guess :wink:

**Pull request merged**